index.html

```diff
@@ -177,7 +177,7 @@ <h2 class="h2">Multi-Post<br>Stories</h2>
     <section class="container-section">
 
       <div class="intro2">
-        <h1>About<br>Myself</h1>
+        <h2>About<br>Myself</h2>
         <p>
           Hello I’m a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience! If
```

Comment:
[optional] Kindly set the `border-top-right-radius`  for this section

![image](https://user-images.githubusercontent.com/43564324/158866393-99249b56-c535-4f31-ae35-2b0d97c43e05.png)

Kindly do the same with `border-bottom-left-radius` for the header section of your page. The aim of the project is to replicate design exactly which would be a great skill for a developer